plugins/docker/README.md

```diff
@@ -0,0 +1,43 @@
+# Docker
```

Comment:
I think it is sufficient to keep all the documentation in the perldoc header of the plugin. Otherwise it could be confusing for people that the README in the directory describes only one of the docker-related plugins in this directory.

Reply:
Removed :)